test/common/http/codec_impl_fuzz_test.cc

```diff
@@ -671,6 +675,9 @@ void codecFuzz(const test::common::http::CodecImplFuzzTestCase& input, HttpVersi
           continue;
         }
       }
+
+      const auto allow_h1_trailers = input.h1_settings().server().enable_trailers() &&
```

Comment:
Please add a TODO noting that we need to cover the asymmetric case.

Reply:
Done